### ISSUE TITLE
add categories via com_categories

### ISSUE THREAD

**Issue by shoulders** [ceford/j4xdemos-com-mywalks#6]
This would be the most helpful feature to be added next.

If it is of any use I went through the whole [Developing an MVC Component (j3.x)](https://docs.joomla.org/J3.x:Developing_an_MVC_Component/Introduction) and built the component. I have changed all of the class names to use the new namespace ones as denoted in **classmap.php**. The code might all be similiar or the same for Joomla 4. The tutorials I found were in depth, I just struggled with the component being called **HelloWorld** and one of the views being called **HelloWorld**, I have corrected this in my version.

- [Classes All Renamed - As per tutorial](releases/tag/1.1.2-rj)

I will be making more changes and improvements to it over time, as you can see I have some issues listed.
- [My Repo](https://github.com/shoulders/com_qwhelloworld-j3/)

**Comment by ceford** [comment]
Last Autumn Astrid did a lot of work on Developing an MVC for J4 Tutorial. It was on Github but has since disappeared. It was massive! Pity!

**Comment by shoulders** [comment]
thats a shame, i would of read it,  was it on github?

perhaps it is in the internetwayback machine

**Comment by ceford** [comment]
Not mine - https://github.com/astridx

**Comment by shoulders** [comment]
is this it? https://github.com/astridx/boilerplate/wiki/000.-Creating-a-Simple-Component-for-Joomla!-4.x---Introduction

I found it in the wiki

**Comment by ceford** [comment]
Yes - that is it. my original link to it was dud and for some reason I did not recognise Boilerplate as the same thing.
